NYTimes.com.js

```diff
@@ -1,224 +1,136 @@
 		}
-
 	}
+}
 
-	associateMeta(newItem, metaTags, "dat", "date");
-	associateMeta(newItem, metaTags, "hdl", "title");
-	associateMeta(newItem, metaTags, "description", "abstractNote");
-	if (type === "blogPost"){
-		associateMeta(newItem, metaTagsProperty, "og:title", "title");
-		associateMeta(newItem, metaTagsProperty, "og:site_name", "blogTitle");
-	}
-	associateMeta(newItem, metaTags, "dsk", "section");
-	associateMeta(newItem, metaTags, "articleid", "accessionNumber");
-	
-	if (newItem.blogTitle) newItem.blogTitle = "The New York Times - " + newItem.blogTitle;
-	
-	if (metaTags["pdate"]) {
-		newItem.date = metaTags["pdate"].replace(/(\d{4})(\d{2})(\d{2})/, "$1-$2-$3");
-	}
 
-	if (metaTags["byl"]) {
-		var author = Zotero.Utilities.trimInternal(metaTags["byl"]);
-		if (author.substr(0, 3).toLowerCase() == "by ") {
-			author = author.substr(3);
+function scrape(doc, url) {
+	var type = detectWeb(doc, url);
+	var translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+	//translator.setDocument(doc);
+	
+	translator.setHandler('itemDone', function (obj, item) {
+		item.language = ZU.xpathText(doc, '//meta[@itemprop="inLanguage"]/@content') || "en-US";
+		if (item.date) {
+			item.date = ZU.strToISO(item.date);
 		}
-
-		var authors = author.split(" and ");
-		for (var i=0; i<authors.length; i++) {
-			var author = authors[i];
-			// fix capitalization
-			var words = author.split(" ");
-			for (var j=0; j<words.length; j++) {
-				words[j] = words[j][0].toUpperCase() + words[j].substr(1).toLowerCase();
-			}
-			author = words.join(" ");
-
-			if (words[0] == "The") {
-				newItem.creators.push({
-					lastName: author,
-					creatorType: "author",
-					fieldMode: true
-				});
-			} else {
-				newItem.creators.push(Zotero.Utilities.cleanAuthor(author, "author"));
+		if (item.itemType == "blogPost") {
+			item.blogTitle = ZU.xpathText(doc, '//meta[@property="og:site_name"]/@content');
+		} else {
+			item.publicationTitle = "The New York Times";
+			item.ISSN = "0362-4331";
+		}
+		//Multiple authors are just put into the same Metadata field
+		//we have to split this here
+		var authors = ZU.xpathText(doc, '//meta[@name="author"]/@content');
+		if (authors) {
+			item.creators = [];
+			var authorsList = authors.split(/,|\band\b/);
+			for (var i=0; i<authorsList.length; i++) {
+				item.creators.push(ZU.cleanAuthor(authorsList[i], "author"));
 			}
 		}
-	}
-
-	if (metaTags["keywords"]) {
-		var keywords = metaTags["keywords"];
-		newItem.tags = keywords.split(",");
-		for (var i in newItem.tags) {
-			newItem.tags[i] = newItem.tags[i].replace("  ", ", ");
+		item.url = ZU.xpathText(doc, '//link[@rel="canonical"]/@href') || url;
+		var pdfurl = ZU.xpathText(doc, '//div[@id="articleAccess"]//span[@class="downloadPDF"]/a[contains(@href, "/pdf")]/@href');
+		if (pdfurl) {
+			item.attachments.push({
+				url: pdfurl,
+				title: "NY Times Archive PDF"
```

Comment:
Okay, I will change this label to "NYTimes Archive PDF" (shorter looks better for me here).